src/mobile/package.json

```diff
@@ -10,13 +10,17 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-navigation/bottom-tabs": "^6.1.0",
+    "@react-navigation/native": "^6.0.7",
+    "@react-navigation/native-stack": "^6.3.0",
```

Comment:
Can we pin these dependencies? (not have the ^ in front of the version). eg `6.1.0` instead of `^6.1.0`

Reply:
sure